ssl/record/methods/ktls_meth.c

```diff
@@ -423,6 +423,9 @@ static int ktls_set_crypto_state(OSSL_RECORD_LAYER *rl, int level,
     if (!BIO_set_ktls(rl->bio, &crypto_info, rl->direction))
         return OSSL_RECORD_RETURN_NON_FATAL_ERR;
 
+    if (rl->direction && (rl->options & SSL_OP_ENABLE_KTLS_TX_ZEROCOPY_SENDFILE))
+        BIO_set_ktls_tx_zerocopy_sendfile(rl->bio);
```

Comment:
Should we check the return value here?

Reply:
No need, it's an opportunistic optimization, it may be unavailable on the running kernel, and the application opting in for it shouldn't get an error in such a case.